src/mame/drivers/superga2.cpp

```diff
@@ -18,16 +18,14 @@
 ************************************************************************/
 
 #include "emu.h"
-#include "video/apple2.h"
 
 #include "cpu/m6502/m6502.h"
-
 #include "machine/74259.h"
 #include "machine/apple2common.h"
 #include "machine/ram.h"
 #include "machine/timer.h"
-
 #include "sound/spkrdev.h"
+#include "video/apple2.h"
```

Comment:
Headers from src/mame should go before headers from src/devices, keeping with the pattern of module header, then most dependent to least dependent module.